tests/functional/s/statement_without_effect.py

```diff
@@ -90,3 +94,7 @@ def unraised_exception():
     """Test that instantiating but not raising an exception is flagged as a pointless statement"""
     ValueError("pointless-statement")  # [pointless-exception-statement]
     ValueError(to_be())  # [pointless-exception-statement]
+    ClassLevelAttributeTest.ClassLevelException(to_be())  # [pointless-exception-statement]
+    ValueError ( "another-pointless-statement" ) # [pointless-exception-statement]
```

Comment:
```suggestion
    ValueError("another-pointless-statement")  # [pointless-exception-statement]
```

Reply:
The code here does look strange, but this was somewhat intentional test coverage to ensure that any string-contents checks used now or in future (like `expr.as_string().endswith(...)`) consider the case that the call name may not be directly adjacent to the parentheses.

Reply:
Had to check, but the spaces are automatically stripped by the Python parser when the code is converted to the AST.